test/e2e/tests/settings/show-native-as-main-balance.spec.js

```diff
@@ -0,0 +1,192 @@
+const { strict: assert } = require('assert');
```

Comment:
We really want to encourage all new tests to be .ts rather than .js, and to use the page object model.  These are not blockers, but if not addressed within this PR would like to request a follow-up to bring the test into compliance with our e2e guidelines [here](https://github.com/MetaMask/contributor-docs/blob/main/docs/e2e/extension-e2e-guidelines.md#page-object-model-pom).  QA teammates are available in metamask-qa slack channel for any questions or support needed.

Reply:
Hi Mike; i have fixed this here [563a536](https://github.com/MetaMask/metamask-extension/pull/27097/commits/563a536eafca604ffa62f560b62d0c5fef065f27)